### ISSUE TITLE
[New Model]: answerdotai/ModernBERT-large

### ISSUE THREAD

**Issue by pooyadavoodi** [vllm-project/vllm#11347]
### The model to consider.

https://huggingface.co/answerdotai/ModernBERT-large

### The closest model vllm already supports.

BAAI/bge-base-en-v1.5

### Before submitting a new issue...

- [ ] Make sure you already searched for relevant issues, and asked the chatbot living at the bottom right corner of the [documentation page](https://docs.vllm.ai/en/latest/), which can answer lots of frequently asked questions.

**Comment by Collen-Roller** [comment]
This would be very helpful!

**Comment by llmforever** [comment]
same here

**Comment by nightflight-dk** [comment]
+1

**Comment by touhi99** [comment]
+1

**Comment by ilkhom19** [comment]
Transformers started supporting it since last week, may be just updating the transformers version would help us ?

https://github.com/huggingface/transformers/releases/tag/v4.48.0

**Comment by ilkhom19** [comment]
https://github.com/vllm-project/vllm/pull/12120

**Comment by geraldstanje** [comment]
whats the status of this? can you already use vllm with modernBERT?

**Comment by pooyadavoodi** [comment]
Not supported yet I think.

I was hoping we could use the fallback transformers backend but that doesn't seem to be compatible for this model due to not supporting flex_attn: https://github.com/vllm-project/vllm/blob/main/vllm/model_executor/model_loader/utils.py#L42

**Comment by pooyadavoodi** [comment]
I haven't used this model yet. Last time I checked, the vllm fallback backend which goes to transformers didn't work.

**Comment by RohitRathore1** [comment]
hi, what's the status on it? Have we started to provide support for this?

**Comment by pooyadavoodi** [comment]
> [@pooyadavoodi](https://github.com/pooyadavoodi) you need transformers version >= 4.49.0. which transformers version was used in your test with vllm?

None of 4.49.0 or tot worked for me. I think the issue is in `is_transformers_impl_compatible` inside vllm as I mentioned a few weeks ago.

**Comment by xsank** [comment]
@pooyadavoodi yes, vllm not support modernBert yet. Only if the transformers support flex attention on modernBert or vllm add modernBert impl it would works.

**Comment by geraldstanje** [comment]
@xsank isnt newest transformers library (transformers version >= 4.49.0.) supporting flex attention already?

**Comment by xsank** [comment]
> [@xsank](https://github.com/xsank) isnt newest transformers library (transformers version >= 4.49.0.) supporting flex attention already? oh looks like its not: [huggingface/transformers#36697](https://github.com/huggingface/transformers/issues/36697)

yes, transformers not supoort yet, i‘m developing the vllm code to support it.
